frontend/pages/hosts/HostDetailsPage/HostDetailsPage.jsx

```diff
@@ -235,12 +236,77 @@ export class HostDetailsPage extends Component {
     );
   };
 
+  renderPacks = () => {
+    const { onPackClick } = this;
+    const { host } = this.props;
+    const { packs = [] } = host;
+
+    const packItems = packs.map((pack) => {
+      return (
+        <li className="list__item" key={pack.id}>
+          <Button
+            onClick={() => onPackClick(pack)}
+            variant="text-link"
+            className="list__button"
+          >
+            {pack.name}
+          </Button>
+        </li>
+      );
+    });
+
+    return (
+      <div className="section section--packs">
+        <p className="section__header">Packs</p>
+        <ul className="list">{packItems}</ul>
+      </div>
+    );
+  };
+
+  renderSoftware = () => {
+    const { host } = this.props;
+    const wrapperClassName = `${baseClass}__table`;
+
+    if (host.software) {
+      return (
+        <div className="section section--software">
+          <div className={baseClass}>
+            <p className="section__header">Software</p>
+            <div className={`${baseClass}__wrapper`}>
+              <table className={wrapperClassName}>
+                <thead>
+                  <tr>
+                    <th>Name</th>
+                    <th>Type</th>
```

Comment:
How bout
```suggestion
                    <th>Source</th>
```
?

I know the mock included a more "pretty" string for this, but I have the API returning just the name of the osquery table the data comes from. 

@noahtalerman if you think it's important to include the pretty type, maybe we can add a small function that maps the source to the pretty type?